SC_redesign/Crop_and_Soil/soil/soil_config_factory.py

```diff
@@ -0,0 +1,39 @@
+from enum import Enum
+import dataclasses
+from SC_redesign.Crop_and_Soil.soil.soil_data import SoilData
+
+"""
+This module serves as a way to easily create and modify SoilData objects based on a set of given configurations. It is 
+based on the CropSpeciesDataFactory model, and should contain all equivalent functionalities.
+"""
+
+
+class SoilConfigurations(Enum):
```

Comment:
I think this should be singular. The instantiated object is a single configuration. 

```suggestion
class SoilConfiguration(Enum):
```

Reply:
Done